<?xml version="1.0"?>
<launch>

    <arg name="distance_between_wheels" default="0.32" /> <!-- CORRECT VALUE HERE -->
    <arg name="wheel_diameter" default="0.066" />

    <!-- base controller, subscribe to cmd_vel (robot body velocity) and publish individual wheel speed -->
    <node pkg="rdl_turtlebot3_navigation" type="base_controller.py" name="turtlebot3_base_controller" output="screen">
        <param name="distance_between_wheels" value="$(arg distance_between_wheels)"/>
        <param name="wheel_diameter"  value="$(arg wheel_diameter)"/>
    </node>

</launch>
